source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -574,6 +586,13 @@ def _set_specific_params(mod):
                 args['_final_line_filter'] = mod.final_line_filter
             if args.get('_max_extra_second_pass') is None:
                 args['_max_extra_second_pass'] = mod.max_extra_second_pass
+
+            args['max_nb_crowfly_by_mode'] = mod.max_nb_crowfly_by_mode # it's a dict of str vs int
+            for mode in args['max_nb_crowfly_by_mode']:
```

Comment:
So if the config is broken, we are never gonna be able to change a mode that's not in config, no ? It's less the case with what you did now in tyr config, I guess ?

Reply:
@pbougue i'll do it in another way